src/Analysis/Ast/Impl/Dependencies/IDependencyChainNode.cs

```diff
@@ -16,6 +16,7 @@
 namespace Microsoft.Python.Analysis.Dependencies {
     internal interface IDependencyChainNode<out TValue> {
         int VertexDepth { get; }
+        bool HasMissingDependencies { get; }
```

Comment:
This property shows if node has any direct or indirect dependencies that aren't added to the graph

Reply:
Add `///` comment?